src/app/page/auth-modal/auth-modal.component.html

```diff
@@ -0,0 +1,38 @@
+<form
+  nz-form
+  [nzLayout]="'inline'"
```

Comment:
А зачем так сложно, можно куда проещ и читаемей:

```suggestion
  nzLayout="inline"
```